File: chatwoot/php/src/ContactUpdateExample.php
<?php

namespace OSEG\ChatwootExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Chatwoot;

$config = Chatwoot\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("api_access_token", "USER_API_KEY");
// $config->setApiKey("api_access_token", "AGENT_BOT_API_KEY");
// $config->setApiKey("api_access_token", "PLATFORM_APP_API_KEY");

$contact_update = (new Chatwoot\Client\Model\ContactUpdate())
    ->setName(null)
    ->setEmail(null)
    ->setPhoneNumber(null)
    ->setAvatarUrl(null)
    ->setIdentifier(null)
    ->setAvatar(null)
    ->setCustomAttributes(null);

try {
    $response = (new Chatwoot\Client\Api\ContactsApi(config: $config))->contactUpdate(
        account_id: 0,
        id: 0,
        data: contact_update,
    );

    print_r($response);
} catch (Chatwoot\Client\ApiException $e) {
    echo "Exception when calling ContactsApi#contactUpdate: {$e->getMessage()}";
}

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/ContactUpdateExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class ContactUpdateExample
{
    fun contactUpdate()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"
        // ApiClient.apiKey["api_access_token"] = "AGENT_BOT_API_KEY"
        // ApiClient.apiKey["api_access_token"] = "PLATFORM_APP_API_KEY"

        val contactUpdate = ContactUpdate()

        try
        {
            val response = ContactsApi().contactUpdate(
                accountId = 0,
                id = 0,
                _data = contactUpdate,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling ContactsApi#contactUpdate")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling ContactsApi#contactUpdate")
            e.printStackTrace()
        }
    }
}
